custom/icds_reports/models/aggregate.py

```diff
@@ -585,7 +626,6 @@ class AggregateComplementaryFeedingForms(models.Model):
     )
 
     class Meta(object):
-        managed = False
```

Comment:
Is this intentional?

Reply:
Yeah, it was django managed, and my bring everything to django pr overcompensated for django issues by accidentally removing that from some of the aggregate tables